File: arc3-api/src/v1/db/Commandstat.js
const mongoose = require('mongoose');

const commandStatSchema = new mongoose.Schema({
    _id: String,
    guild_id: String,
    args: Object,
    command_name: String
}, {
    collection: "Commandstats"
})

const CommandStat = mongoose.model("Commandstat", commandStatSchema);

module.exports = CommandStat;

File: Arc3/Core/Schema/CommandStat.cs
using System.Collections;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace Arc3.Core.Schema;

public class CommandStat {

    [BsonId]
    [BsonRepresentation(BsonType.String)]
    public string Id { get; set; }

    [BsonElement("guild_id")]
    public string GuildID { get; set; }

    [BsonElement("args")]
    public BsonDocument Args { get; set; }

    [BsonElement("command_name")]
    public string Name { get; set; }

}